interfaces/builtin/audio_record.go

```diff
@@ -0,0 +1,82 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/interfaces/apparmor"
+	"github.com/snapcore/snapd/snap"
+)
+
+// The audio-record interface is the companion interface to the audio-playback
+// interface and is not meant to be used without it. The design of this
+// interface is based on the idea that the slot implementation (eg pulseaudio)
+// is expected to query snapd on if the audio-record slot is connected or not
+// and the audio service will mediate recording (ie, the rules below allow
+// connecting to the audio service, but do not implement enforcement rules; it
+// is up to the audio service to provide enforcement). If other audio recording
+// servers require different security policy for record (eg, a different socket
+// path), then those accesses will be added to this interface.
+
+const audioRecordSummary = `allows operating as or interacting with recording audio services`
```

Comment:
same comment as niemeyer made on the playback one.

Reply:
Done. I've added a card to add spread tests so this is not forgotten.